agent/util-scripts/pbench-copy-result-tb

```diff
@@ -65,6 +65,8 @@ fi
 # foo.tar.xz.md5.check file to foo.tar.xz.md5. That's the signal that the
 # agent has finished with this tarball.
 
+controller=$(basename ${controller_dir})
+ssh -n -i ${pbench_bin}/id_rsa ${ssh_opts} ${results_repo} "mkdir -p ${results_path_prefix}/${controller}"
```

Comment:
This is probably beyond the scope of this PR, but we should really consider using the `-F` switch to `ssh`, which would likely allow us to avoid having to specify things like `-i` and the other `${ssh_opts}` here.  This would make the scripts' command lines simpler as well as allowing us to do things like adding options (e.g., for persistent sessions) in a single, centralized place, rather than having to go and change each individual `ssh` invocation.

Reply:
Tracked in #2567.